helpers/faq.py

```diff
@@ -0,0 +1,143 @@
+import discord
+
+
+async def send_faq_embed(channel, user_mention, topic, description):
+    await channel.send(f"{user_mention} I believe you're asking about {topic}:")
+    embed = discord.Embed(description=description, color=discord.Color(0x43C6E7))
+    await channel.send(embed=embed)
```

Comment:
thought(non-blocking): this belongs more in the cog itself, as there's nothing else here but static content, but either approach is fine.